### PR TITLE
Enable webworkers when using a non-tls connection

### PR DESCRIPTION
**Pull request by pe4cey**
- Allows non-blocking rendering when executing a long running cypher query

### CODE REVIEW

File: src/shared/services/commandInterpreterHelper.js [hunk 1]
```diff
@@ -133,11 +134,18 @@ const availableCommands = [
     match: cmd => /^cypher$/.test(cmd),
     exec: (action, cmdchar, put, store) => {
       const state = store.getState()
+
+      console.log(
```
Comment (oskarhane) [inline src/shared/services/commandInterpreterHelper.js]: *cough*

Comment (oskarhane) [inline src/shared/services/commandInterpreterHelper.js]: *cough*

**Comment by pe4cey** [comment]
Updated

**Comment by oskarhane** [comment]
Closing this in favour of https://github.com/neo4j/neo4j-browser/pull/670
I think we hit https://bugs.chromium.org/p/chromium/issues/detail?id=760708 when it failed.

**Pull request closed without merging**